cmd/todos_server/main.go

```diff
@@ -1,21 +1,25 @@
 package main
 
 import (
-	"github.com/go-chi/chi"
 	"log"
 	"net/http"
 
-	impl "github.com/anz-bank/syslgen-examples/todos-impl"
+	"github.com/go-chi/chi"
+
+	impl "github.com/anz-bank/syslgen-examples/pkg/todos-impl"
 
-	"github.com/anz-bank/syslgen-examples/todos"
+	"github.com/anz-bank/syslgen-examples/gen/todos"
 )
 
+const serverPort = ":8080"
+
 func main() {
 	serviceImpl := impl.NewServiceImpl()
 	svcHandler := todos.NewServiceHandler(serviceImpl)
 	serviceRouter := todos.NewServiceRouter(svcHandler)
 	router := chi.NewRouter()
 	serviceRouter.Route(router)
 
-	log.Fatal(http.ListenAndServe(":8080", router))
+	log.Println("Starting Server on localhost" + serverPort)
+	log.Fatal(http.ListenAndServe(serverPort, router))
```

Comment:
Right now, this listens on "0.0.0.0:8080" (all interfaces) not "localhost:8080" (usually 127.0.0.1 and maybe also ::1).

Reply:
@anzdaddy Fixed up as per review comments, ready for re-revie and merge!